apps/designer/app/designer/shared/assets/use-assets.ts

```diff
@@ -93,12 +101,28 @@ export const useAssets = (type: AssetType) => {
           toast.error(error.message);
         }
       });
+
+    for (const formData of formsData) {
+      submit<UploadData>(
+        formData,
+        {
+          method: "post",
```

Comment:
In all RFCs `POST` `GET` uppercased

Reply:
Yeah, it gets converted I believe in the end, but since it works, in the end it will become a mixture anyways, to enforce it more or less we probably need to abstract it away with a const or something or just ignore it and live with inconsistency

Reply:
checked remix docs, even they are inconsistent

Reply:
it should have never worked both ways to be consistent, I suggest just to ignore it

Reply:
in HTML lowercase is allowed https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

Reply:
actually its not just allowed in html, its how its suggested